### PR TITLE
HNL trilepton, ele and mu couplings

### PR DESCRIPTION
**Pull request by cecilecaillol**
HNL trilepton, with ele and mu couplings.
The cards are similar to what was just merged for taus. 

There are 4 directories: 2016 and 2017/2018 for electron or muon couplings.
In each directory there are 3 phases spaces: mN < mW = LO with 3-body decay, mW < mN < mZ = NLO with decays to W, mN > mZ = NLO, decays to W and Z

### CODE REVIEW

**Comment by sihyunjeon** [comment]
@cecilecaillol 
Hello,

So here, the only changes from tau cards is,
you turned on el(mu) coupling only and your n1 is attached to el(mu) instead of taus.
Is this correct?

**Comment by cecilecaillol** [comment]
Yes, I have changed the couplings in customize_cards, and the leptons in the decays in proc_card. The rest is identical.

**Comment by sihyunjeon** [comment]
@agrohsje The cards are fine for me

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNgtmWltmZ_V-0p01_e_NLO/HeavyNeutrino_trilepton_V-0p01_e_NLO_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+define l+ = e+
+define ldecay+ = e+ mu+ ta+
+define l- = e-
+define ldecay- = e- mu- ta-
+define l = l+ l-
+define ldecay = ldecay+ ldecay-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define v = vl vl~
+define w = w- w+
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNgtmWltmZ_V-0p01_e_NLO/HeavyNeutrino_trilepton_V-0p01_e_NLO_proc_card.dat:20]: wouldn't it be cleaner to remove all this info and define the particles you really need only after you imported your model?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNgtmZ_V-0p01_e_NLO/HeavyNeutrino_trilepton_V-0p01_e_NLO_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+define l+ = e+
+define ldecay+ = e+ mu+ ta+
+define l- = e-
+define ldecay- = e- mu- ta-
+define l = l+ l-
+define ldecay = ldecay+ ldecay-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define v = vl vl~
+define w = w- w+
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNgtmZ_V-0p01_e_NLO/HeavyNeutrino_trilepton_V-0p01_e_NLO_proc_card.dat:20]: As above.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNltmW_V-0p01_e_LO/HeavyNeutrino_trilepton_V-0p01_e_LO_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+define l- = e-
+define ldecay- = e- mu- ta-
+define l = l+ l-
+define ldecay = ldecay+ ldecay-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define v = vl vl~
+define w = w- w+
+define p = p b b~
+define j = p
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNltmW_V-0p01_e_LO/HeavyNeutrino_trilepton_V-0p01_e_LO_proc_card.dat:22]: Same here.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNltmW_V-0p01_e_LO/HeavyNeutrino_trilepton_V-0p01_e_LO_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+  0.0	=  polbeam1  ! beam polarization for beam 1
+  0.0	=  polbeam2  ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  lhapdf        =  pdlabel    ! PDF set
+  262000        =  lhaid      ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNltmW_V-0p01_e_LO/HeavyNeutrino_trilepton_V-0p01_e_LO_run_card.dat:38]: Sorry I should have already noticed before. Is it on prupose to use LO PDF with alphas=0.118. Usually the higher value of 0.130 is used to compensate for the missing contributions. Just double-checking.

**Comment by agrohsje** [comment]
My main comment is about alphas in LO PDF and then the clean-up of the process cards. This is a bit hard to read. The PDF comment might also apply to the other cards we already merged. Sorry for not spotting before.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNltmW_V-0p01_e_LO/HeavyNeutrino_trilepton_V-0p01_e_LO_run_card.dat (hunk 4 above)
Comment (sihyunjeon) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNltmW_V-0p01_e_LO/HeavyNeutrino_trilepton_V-0p01_e_LO_run_card.dat:38]: Ah I think I recommended that PDF a bit while ago.
Will it give unignorable differences to LO process?

Comment (agrohsje) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNltmW_V-0p01_e_LO/HeavyNeutrino_trilepton_V-0p01_e_LO_run_card.dat:38]: In 2016, we typically used 0.130 with LO PDF. The argument is that the higher value at mZ makes up for part of what we are missing from using LO PDF. I cannot really comment on the modeling or detailed impact. It is more like a hand waving argument people are usually using.

**Comment by sihyunjeon** [comment]
@agrohsje I'll ask the analysers to check what the impact is at least on the cross sections.

**Comment by sihyunjeon** [comment]
@cecilecaillol a reminder for you. already asked Liam as well

**Comment by sihyunjeon** [comment]
@agrohsje They decided to move up to UL instead of using the old prelegacy. So 2016 should be cleaned up, I'll ask them to do so.

**Comment by menglu21** [comment]
@cecilecaillol ping

**Pull request closed without merging**